### DISCUSSION TITLE
DietPi-Software | Homebridge

### DISCUSSION THREAD

**Discussion by holocronology** [MichaIng/DietPi#5142] [Software Request] [answered]
Request to add Homebridge to Dietpi-Software install list.

**Comment by MichaIng** [comment]
Many thanks for your request.

While FeatHub is dead, there was a request already with 60 votes in ~3 years, hence top 7 software request: https://feathub.com/MichaIng/DietPi/+51

Please (re)place your vote via arrow up button on above OP post.

**Reply by Joulinar** [reply, marked as answer]
PR up https://github.com/MichaIng/DietPi/pull/6493
